cypress/integration/pages/testsForAllCanonicalPages.js

```diff
@@ -16,6 +19,23 @@ export const testsThatFollowSmokeTestConfigForAllCanonicalPages = ({
 }) => {
   if (pageType !== 'errorPage404') {
     describe(`Running testsForAllCanonicalPages for ${service} ${pageType}`, () => {
+      describe('ATI', () => {
```

Comment:
Should this be in the if not error page?

Reply:
I'm not sure, but before I started working on this there were no ATI tests in the errorPage404 test files, so that's why I placed them in the if not error page block for all tests.